masterlist.yaml

```
@@ -8307,6 +8307,10 @@ plugins:
       - link: 'https://www.nexusmods.com/oblivion/mods/44440/'
         name: 'Master Quest Arcturus'
     req: [ 'DLCShiveringIsles.esp' ]
+    msg:
+      - <<: *compatIssuesWithX
```

This mod doesn't have compatibility notes, so `*patchUnavailable` would be more appropriate.

```suggestion
      - <<: *patchUnavailable
```